tests/global_functions.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * @author  Andreas Heigl<andreas@heigl.org>
+ * @license http://www.wtfpl.net/txt/copying/ WTFPL
+ * @since   08.02.15
+ */
+function emptyFunc()
+{
+    // Empty Function as required by codesniffer instead of an empty anonymous
+    // function
+}
```

Comment:
I'm using PHPCS 2.2.0. I did a ```composer require squizlabs/php_codesniffer``` to install codesniffer and then ran ```./vendor/bin/phpcs --standard=PSR2 classes/ tests/```. That failed (with a warning) due to the empty function in ```./tests/MockTest:172``` where I had originally ```$mock = new Mock(__NAMESPACE__, $name, function(){});```.